fiftyone/core/collections.py

```diff
@@ -9069,15 +9084,19 @@ def get_index_information(self, include_size=False):
         fields_map = self._get_db_fields_map(reverse=True)
         sample_info = self._dataset._sample_collection.index_information()
 
+        sample_cs = None
         if include_size:
-            conn = foo.get_db_conn()
-            cs = conn.command(
-                "collstats", self._dataset._sample_collection_name
-            )
-            for key, size in cs["indexSizes"].items():
+            sample_cs = self._sample_stats()
+            for key, size in sample_cs["indexSizes"].items():
                 if key in sample_info:
                     sample_info[key]["size"] = size
 
+        if include_progress:
+            sample_cs = sample_cs or self._sample_stats()
+            for key in sample_cs["indexBuilds"]:
```

Comment:
Currently, we only set `in_progress=True` for the index that is being created and skip it for already built indexes; we can also set this field `in_progress=False` for the built indexes as well.